detection_rules/attack.py

```diff
@@ -105,16 +105,16 @@ def get_version_from_tag(name, pattern='att&ck-v'):
         _, version = name.lower().split(pattern, 1)
         return version
 
-    current_version = get_version_from_tag(filename, 'attack-v')
+    current_version = VersionInfo(*get_version_from_tag(filename, 'attack-v').split("."))
```

Comment:
Once we update the package version we can do this:

```suggestion
    current_version = Version(get_version_from_tag(filename, 'attack-v'), optional_minor_and_patch=True)
```

Reply:
Great call! I updated the code to be `Version.parse(VERSION, optional_minor_and_Patch=True)` where necessary.